core/server/master/src/main/java/alluxio/master/throttle/SystemMonitor.java

```diff
@@ -390,14 +410,16 @@ private void checkAndBackPressure() {
       if (mDeltaFilesystemIndicators != null) {
         LOG.warn("The delta filesystem indicators {}", mDeltaFilesystemIndicators);
       }
-    }
+    } else {
+      LOG.debug("The system status is {}, now in {}, related Server aggregate indicators:{},"
+              + " pit indicators:{}",
+          statusTransition, mCurrentSystemStatus, mAggregatedServerIndicators, pitIndicator);
 
-    LOG.debug("The system status is {}, now in {}, related Server aggregate indicators:{},"
-            + " pit indicators:{}",
-        statusTransition, mCurrentSystemStatus, mAggregatedServerIndicators, pitIndicator);
+      LOG.debug("The delta filesystem indicators {}",
```

Comment:
check null before the LOG.debug?
if (mDeltaFilesystemIndicators != null){
....
}